src/Projection/ProjectionValidatorException.php

```diff
@@ -0,0 +1,14 @@
+<?php declare(strict_types=1);
+
+namespace Comquer\Projection;
+
+use RuntimeException;
+
+class ProjectionValidatorException extends RuntimeException
+{
+    public static function missingRequiredKeysInSerializedProjection(string $projectionName, array $missingKeys): self
+    {
+        $missingKeys = implode(", ", $missingKeys);
```

Comment:
Minor: use single quotations where interpolation is not needed (consider this as a micro-optimisation)